### ISSUE TITLE
How do I twilio?

### ISSUE THREAD

**Issue by kbruegge** [fact-project/pycustos#1]
Hello!

```
$ python examples/twilio.py 
Example started
Traceback (most recent call last):
  File "examples/twilio.py", line 31, in <module>
    level=levels.INFO,
  File "/Users/Kai/anaconda3/lib/python3.5/site-packages/custos/notify/twilio.py", line 41, in __init__
    raise ImportError('The twilio package is required for this Notifier')
ImportError: The twilio package is required for this Notifier
```

Do I need another package installed?

**Comment by maxnoe** [comment]
Yes, I did not find a nicer way of using optional dependencies with python.

`pip install twilio`, same is valid for the telegram bot. You will need the `telepot` package for that.

**Comment by kbruegge** [comment]
I must be doing something wrong here.

```

Kai@MacBook-Pro ~/Development/pycustos (master) 
$ pip install twilio --upgrade
Collecting twilio
Requirement already up-to-date: pysocks in /Users/Kai/anaconda3/lib/python3.5/site-packages (from twilio)
Requirement already up-to-date: six in /Users/Kai/anaconda3/lib/python3.5/site-packages (from twilio)
Collecting pytz (from twilio)
  Using cached pytz-2016.6.1-py2.py3-none-any.whl
Requirement already up-to-date: httplib2>=0.7 in /Users/Kai/anaconda3/lib/python3.5/site-packages (from twilio)
Installing collected packages: pytz, twilio
  Found existing installation: pytz 2015.7
    DEPRECATION: Uninstalling a distutils installed project (pytz) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.
    Uninstalling pytz-2015.7:
      Successfully uninstalled pytz-2015.7
Successfully installed pytz-2016.6.1 twilio-5.4.0
You are using pip version 8.1.0, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Kai@MacBook-Pro ~/Development/pycustos (master) 
$ python examples/twilio.py 
Example started
Traceback (most recent call last):
  File "examples/twilio.py", line 31, in <module>
    level=levels.INFO,
  File "/Users/Kai/anaconda3/lib/python3.5/site-packages/custos/notify/twilio.py", line 41, in __init__
    raise ImportError('The twilio package is required for this Notifier')
ImportError: The twilio package is required for this Notifier
```

**Comment by kbruegge** [comment]
Fixed in commit 8993b354e7c1e03c35c6e552edaf444527cf8179
